### PR TITLE
Layout all pages according design

### PR DESCRIPTION
**Pull request by BogdanParashchak**

### CODE REVIEW

File: src/component/Common/FormsControl/FormControl.module.css [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+  border: #ff480b solid 4px;
 }
 
 .textError{
   color: #ff480b;
+  font-size: 0.75em;
+  padding-top: 0.3em;
+  line-height: 1em;
+  display: block;
+}
```
Comment (TarasOrlovskyi) [inline src/component/Common/FormsControl/FormControl.module.css:15]: I think in we can put width: 100% in this class in order to error message will be flexible

File: src/component/Contact/ContactUs.module.css [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
+    /*height: 30.51em;*/
+    height: 27em;
+  }
+
+  .contactUsPageUser {
+    /*height: 24.17em;*/
+    height: 20.83em;
+  }
+
+
```
Comment (TarasOrlovskyi) [inline src/component/Contact/ContactUs.module.css:31]: delete empty space

File: src/component/Contact/ContactUsForm/ContactUsForm.jsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ class ContactUsForm extends React.Component {
       <form>
         <div className={userStyle.user + ' ' + contactStyle.contactUsPageUser}>
           <div className={userStyle.user__fieldsWrapper}>
-            <div className={userStyle.user__fields + ' ' + contactStyle.contactUsPageUser__fields}>
+            <div className={userStyle.user__fields/* + ' ' + contactStyle.contactUsPageUser__fields*/}>
```
Comment (TarasOrlovskyi) [inline src/component/Contact/ContactUsForm/ContactUsForm.jsx:23]: delete all comments

File: src/component/RegistrationPages/SignIn/SignIn.module.css [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
 
 .signInUser {
-  height: 14.39em;
+ height: 13.89em;
 }
 
 .signInUser__fields {
-  height: 9em;
+
 }
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/SignIn.module.css:19]: you should delete empty class

File: src/component/RegistrationPages/SignIn/SignIn.module.css [hunk 5]
```diff
@@ -26,24 +25,31 @@
 
 @media (max-width: 768px) {
   .contentSignIn {
-    height: 447px;
-    padding: 70px 0;
+   /* height: 447px;
+    padding: 70px 0;*/
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/SignIn.module.css:29]: delete comments

File: src/component/RegistrationPages/SignIn/SignIn.module.css [hunk 6]
```diff
@@ -26,24 +25,31 @@
...
   .contentSignIn__column {
-    height: 447px;
+    height: 452px;
     width: 100%;
+    max-width: 17.22em;
   }
+
   .signInUser {
-    height: 225px;
+    /*height: 17.78em;*/
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/SignIn.module.css:40]: delete all comments

File: src/index.css [hunk 7]
```diff
@@ -192,6 +192,7 @@ main {
   align-items: center;
 }
 
+
```
Comment (TarasOrlovskyi) [inline src/index.css:195]: delete empty space

**Review by TarasOrlovskyi — CHANGES_REQUESTED** [review]

File: src/component/Contact/ContactUs.module.css (hunk 2 above)
Comment (BogdanParashchak) [reply src/component/Contact/ContactUs.module.css:31]: done

File: src/component/Contact/ContactUsForm/ContactUsForm.jsx (hunk 3 above)
Comment (BogdanParashchak) [reply src/component/Contact/ContactUsForm/ContactUsForm.jsx:23]: done

File: src/component/RegistrationPages/SignIn/SignIn.module.css (hunk 4 above)
Comment (BogdanParashchak) [reply src/component/RegistrationPages/SignIn/SignIn.module.css:19]: done

File: src/component/RegistrationPages/SignIn/SignIn.module.css (hunk 5 above)
Comment (BogdanParashchak) [reply src/component/RegistrationPages/SignIn/SignIn.module.css:29]: done

File: src/component/RegistrationPages/SignIn/SignIn.module.css (hunk 6 above)
Comment (BogdanParashchak) [reply src/component/RegistrationPages/SignIn/SignIn.module.css:40]: done

File: src/index.css (hunk 7 above)
Comment (BogdanParashchak) [reply src/index.css:195]: done

File: src/component/Common/FormsControl/FormControl.module.css (hunk 1 above)
Comment (BogdanParashchak) [reply src/component/Common/FormsControl/FormControl.module.css:15]: done

**Review by TarasOrlovskyi — APPROVED** [review]

**Pull request merged**